experiments/dataSources/Salesforce_REST/readme.md

```diff
@@ -0,0 +1,50 @@
+## Salesforce data access using REST
+Salesforce has a super rich array of APIs that provide access to far more data than can be gathered from Skuid's direct SFDC data source.  
+
+Some examples of this include: 
+
+* Access to Data.com objects
+* Access to the [interface API](https://developer.salesforce.com/docs/atlas.en-us.uiapi.meta/uiapi/ui_api_get_started.htm)-  like creating a model of field picklist values. 
+* Access to many [setup objects](https://developer.salesforce.com/docs/atlas.en-us.220.0.object_reference.meta/object_reference/sforce_api_objects_concepts.htm) not exposed as objects to Skuid.  
+
+Fortunately it is easy to build a REST data source that accesses Salesforce so you can use these other APIs and access these objects. 
+
+## How to use this data source
+
+### Create the data source
+
+First you'll need to create a new REST data source that connects to the target Salesforce org.  Configure it as follows:
+* URL / Endpoint: ``https://<<MyDomain>>.my.salesforce.com``
+* Use Proxy:  False
+* Authentication Method:  **No authentication**
```

Comment:
If we're going to bold any parts of the settings, it should be the setting label, not the value.
```suggestion
* Authentication Method:  No Authentication
```